todo/src/App.js

```diff
@@ -1,21 +1,141 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+import React from "react";
+import TodoList from "./components/TodoList";
+import TodoForm from "./components/TodoForm";
+import { connect } from 'react-redux';
 
-class App extends Component {
+class App extends React.Component {
```

Comment:
How you initially had line 1, importing `{ Component }` was great, then you can keep below syntax
```
class App extends Component {}
```
Slightly cleaner 👍